src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -465,7 +459,7 @@ private void CloseInner()
             while (!acceptOrConnectQueue.IsEmpty)
             {
                 AcceptOrConnectOperation op = acceptOrConnectQueue.Head;
-                bool completed = op.TryCompleteAsync(_fileDescriptor);
+                bool completed = op.TryCompleteAsync(_fileDescriptor, true);
```

Comment:
Consider naming the Boolean argument to make it easier to read at the call site, e.g. ```forceComplete: true``` instead of just ```true```.